optuna/samplers/_brute_force.py

```diff
@@ -190,9 +190,10 @@ def sample_independent(
                 TrialState.FAIL,
             ),
         )
-        tree = self._build_tree((t for t in trials if t.number != trial.number), trial.params)
+        tree = _TreeNode()
         candidates = _enumerate_candidates(param_distribution)
         tree.expand(param_name, candidates)
+        self._populate_tree(tree, (t for t in trials if t.number != trial.number), trial.params)
```

Comment:
I added this comment for future modifications.
```suggestion
        # Populating must happen after the initialization above to prevent tree from
        # being initialized as an empty graph, which is created n_jobs > 1
        # where we get trials.params = {}.
        self._populate_tree(tree, (t for t in trials if t.number != trial.number), trial.params)
```